test/bdd/fixtures/auth-rest/docker-compose.yml

```diff
@@ -14,10 +14,66 @@ services:
       - AUTH_REST_HOST_URL=0.0.0.0:8070
       - AUTH_REST_TLS_CACERTS=/etc/tls/ec-cacert.pem
       - AUTH_REST_TLS_SYSTEMCERTPOOL=true
-      - DATABASE_TYPE=mem
+      - AUTH_REST_TLS_SERVE_CERT=/etc/tls/ec-pubCert.pem
+      - AUTH_REST_TLS_SERVE_KEY=/etc/tls/ec-key.pem
+      - AUTH_REST_DATABASE_TYPE=mem
+      - AUTH_REST_OIDC_CALLBACK=TODO
+      - AUTH_REST_GOOGLE_URL=http://mock.oidc.provider
+      - AUTH_REST_GOOGLE_CLIENTID=TODO
+      - AUTH_REST_GOOGLE_CLIENTSECRET=TODO
     ports:
       - 8070:8070
     entrypoint: ""
-    command:  /bin/sh -c "auth-rest start"
+    command:  /bin/sh -c "sleep 10 && auth-rest start"
```

Comment:
It would probably be nice to have a followup issue created to avoid the hardcoded sleep

Reply:
@DRK3 will put a TODO

By the way - this sleep time is less than the BDD [`testSleep`](https://github.com/trustbloc/hub-auth/blob/b89a052c36265fbbea4c14b724c8c7c4014c95d3/test/bdd/bddtests_test.go#L69).

Reply:
@DRK3 done